library/blivet.py

```diff
@@ -411,6 +413,11 @@ def _reformat(self):
         """ Schedule actions as needed to ensure the volume is formatted as specified. """
         fmt = self._get_format()
         if self._device.format.type == fmt.type:
+            # format is the same, not need to run reformatting
+            if self._device.format.label != fmt.label and fmt.label != '':
```

Comment:
We should probably allow clearing/removing a label, no?

Reply:
This is a workaround. At this moment updating label value to empty string causes blivet to fail. Unfortunately empty string is the default value and it causes failures within storage role tests.

Reply:
The root cause this error was in another test that tried to modify label of mounted xfs system. Workaround was removed and empty string can be now used to remove existing label.